docs/usage/model_config.md

```diff
@@ -115,6 +115,9 @@ not be included in the model schemas. **Note**: this means that attributes on th
 **`copy_on_model_validation`**
 : whether or not inherited models used as fields should be reconstructed (copied) on validation instead of being kept untouched (default: `True`)
 
+**`partial`**
```

Comment:
I let you see with the author of the other PR and others for the best wording (`total` was my suggestion to mimic `TypedDict` but that's just an idea)

Reply:
Yeah I'm quite flexible on phrasing. I was basing it off of Django REST Framework as that's where I'm coming to FastAPI / pydantic from.

Reply:
I agree on "partial", because:
1. It matches (roughly) TypeScript which is kind of the gold standard for type hints in dynamic languages
2. It means you can talk about "partial models" which is cleaner than "non-total models"

Reply:
But can we add a note to the docs explaining that the word "total" could also be used with the opposite meaning.